config/regex.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
     TEMPLATE_NAME: /^[\w/-]+$/,
     // Version can only have up to 2 decimals, like 1.2.3
     VERSION: /^(\d+\.)?(\d+\.)?(\d+)$/,
```

Comment:
Idk what is using `VERSION` so I don't want risk breaking whatever that's using it.